decode.go

```diff
@@ -412,6 +412,23 @@ func (bidm BigIntDecMode) valid() bool {
 	return bidm >= 0 && bidm < maxBigIntDecMode
 }
 
+// ByteStringToStringMode specifies the behavior when decoding a CBOR byte string into a Go string.
+type ByteStringToStringMode int
+
+const (
+	// ByteStringToStringError generates an error on an attempt to decode a CBOR byte string into a Go string.
+	ByteStringToStringError ByteStringToStringMode = iota
+
+	// ByteStringToStringAllow permits decoding a CBOR byte string into a Go string.
+	ByteStringToStringAllow
```

Comment:
Maybe rename to use similar naming convention as existing options like `MapKeyByteStringMode`.

```suggestion
	// ByteStringToStringForbidden forbids decoding CBOR byte string into Go string by
	// returning UnmarshalTypeError.
	ByteStringToStringForbidden ByteStringToStringMode = iota

	// ByteStringToStringAllowed allows decoding a CBOR byte string into a Go string.
	ByteStringToStringAllowed
```